test/utils/TestUtils.hs

```diff
@@ -164,8 +168,7 @@ logFilePath = "functional-hie-" ++ stackYaml ++ ".log"
 -- We also need to unset STACK_EXE manually inside the tests if they are
 -- run with `stack test`
 hieCommand :: String
-hieCommand = "stack exec --no-stack-exe --no-ghc-package-path --stack-yaml=" ++ stackYaml ++
-             " hie -- -d -l test-logs/" ++ logFilePath
+hieCommand = "hie -d -l test-logs/" ++ logFilePath
```

Comment:
We need to run hie via stack in order to get the right version we’re testing

Reply:
Yes, I committed that accidentally! Will be fixed ASAP, thanks!